projects/final-project/planning-your-project.md

```diff
@@ -94,6 +95,46 @@ Look at conventions on other sites and stick with them. For example, login links
 - [ ] Revise your plan based on what you’ve learned and the decisions you’ve made. Do you need to eliminate or adjust your data model, wireframes, or most-important features?
 - [ ] Put your content, pages, features, and design tasks into order of priority.
 
+### Timeline
+
+Take a look at the required milestones for all four weeks of your project and use them when formulating your proposed project schedule. The milestones are as follows:
+
+**Week 1 Milestones**
+[ ] Fork the project template.
+[ ] Deploy your initial project files.
+[ ] Connect and test your database and APIs.
+[ ] Initialize the components.
+[ ] Have 1 code review and PR with your mentor and 1 PR partner.
+[ ] Make sure you're committing at least 3 times and pushing at least 1 time per day.
+
+
+**Week 2 Milestones**
+- [ ] Create your tests (unit and integration, if possible!).
+- [ ] Complete MVP functionality.
+- [ ] Re-scope if needed.
+- [ ] Your medium article text will be due on Friday - make sure your README has been updated and you are ready to talk about your project!
+- [ ] Complete all primary features.
+- [ ] Launch your project to Render.
+- [ ] Have 1 code review and PR with your mentor and 1 PR partner.
+
+**Week 3 Milestones**
+- [ ] Make a "complete_mvp" branch.
+- [ ] Build any nice-to-have/secondary features you have time for.
+- [ ] Polish the visual design and check for accessibility and UI/UX best practices.
+- [ ] Prepare your personal pitch and project demo (<3 minutes).
+- [ ] Refactor your code.
+- [ ] Check that your variable and function names are self-explanatory.
+- [ ] Clarify your code comments.
+- [ ] Add setup instructions to your repo's README, and make any updates so it reflects the current state of the app. Don't forget a screenshot!
+- [ ] Have 1 code review and PR with your mentor and 1 PR partner.
+- [ ] Ensure you have at least 50 commits (closer to 200 is better).
+
+**Week 4 Milestones**
+- [ ] Present final project to cohort!
+- [ ] Record and submit your personal pitch and demo video—it should be 2 minute MAX for each part with NO READING
+
+- [ ] Create a daily project schedule based on the weeks 1-4 outlines. Week 4 can be used for stretch goals if time allows.
+
```

Comment:
Let's remove the list of milestones here and directly refer to the Schedule section of [the main README.md](https://github.com/Techtonica/curriculum/blob/main/projects/final-project/README.md#schedule)

If there are any points in these milestones that were not in the main README, please update those lists

Reply:
Removed the list and linked to the schedule section!

Reply:
If any of the removed milestone content that was previously in this PR was not referenced in the milestones that lives in the main README (for example perhaps the google doc milestones say something different from the README by having an extra bulleted requirement) than please ensure that the main README where the milestones will ultimately live are all inclusive and not missing anything from the google doc.

Reply:
Thanks for the clarification! I have added the things I observed were missing from the main readme.